modules/compliance-tests/test/src-ce2/smithy4s/compliancetests/WeaverComplianceTest.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.compliancetests
```

Comment:
Why do we have two of the same file? Given we have Compat outsourced, could we just have one `src/` implementation of `WeaverComplianceTest`?

Reply:
I've changed a lot of things over the lifetime of this PR, at some point I was extending `SomeClassThatNeededCompat` and I was not able to implement it w/o having essentially the same file in the two source trees. But now that's gone, I'm pretty sure you're right and this can be moved to the main source tree